packages/terra-demographics-banner/src/DemographicsBanner.scss

```diff
@@ -1,103 +1,84 @@
-@import '~terra-mixins';
 @import './variables';
 
-.terra-DemographicsBanner {
-  background: $demographics-background;
-  color: $demographics-foreground;
-  margin-bottom: 0.714em;
-  padding: 0.714em;
+/* stylelint-disable selector-class-pattern  */
 
-  @media screen and (min-width: 768px) {
-    display: flex;
-    flex-direction: row;
-  }
-}
-
-.terra-DemographicsBanner-value {
-  display: inline-flex;
-
-  @if $terra-bidi {
-    [dir='ltr'] & {
-      margin-right: 0.357em;
-
-      &:last-child {
-        margin-right: 0;
-      }
-    }
-
-    [dir='rtl'] & {
-      margin-left: 0.357em;
+:local {
+  .demographics-banner {
+    background: var(--terra-demographics-banner-background);
+    color: var(--terra-demographics-banner-foreground);
+    margin-bottom: 0.714em;
```

Comment:
Are there any requirements around needing this margin-bottom? While building components this causes the margin to create noticeable gaps in content